### ISSUE TITLE
POS / parsing

### ISSUE THREAD

**Issue by githubnemo** [kratenko/deepmud#1]
spaCy sieht ganz gut aus:

https://spacy.io/docs/usage/pos-tagging#pos-tagging-german

**Comment by kratenko** [comment]
Jo, sieht gut aus. Hab nur Beschreibung dazu gelesen, nichts ausprobiert. Käme auch auf ein konkretes Problem an, das man mal damit lösen sollte. Fühl dich frei, du bist der, der hier am meisten Ahnung von Sprachverarbeitung hat.

**Comment by githubnemo** [comment]
"Ahnung" sagt er. OK ich guck was ich damit anstellen kann.

**Comment by kratenko** [comment]
Bin gespannt auf deine Ergebnisse. Klingt spannend, aber ich wüsst noch nicht, wo ich anfangen würde, damit was zu machen...
